public/javascripts/SVLabel/build/SVLabel.js

```diff
@@ -11735,39 +11747,207 @@ function ModalMissionComplete ($, d3, L) {
 
     function _updateMissionProgressStatistics (auditedDistance, missionDistance, remainingDistance, unit) {
         if (!unit) unit = "kilometers";
-        svl.ui.modalMissionComplete.totalAuditedDistance.html(auditedDistance.toFixed(2) + " " + unit);
-        svl.ui.modalMissionComplete.missionDistance.html(missionDistance.toFixed(2) + " " + unit);
-        svl.ui.modalMissionComplete.remainingDistance.html(remainingDistance.toFixed(2) + " " + unit);
+        svl.ui.modalMissionComplete.totalAuditedDistance.html(auditedDistance.toFixed(1) + " " + unit);
+        svl.ui.modalMissionComplete.missionDistance.html(missionDistance.toFixed(1) + " " + unit);
+        svl.ui.modalMissionComplete.remainingDistance.html(remainingDistance.toFixed(1) + " " + unit);
+    }
+    
+    // converts GeoJSON from the LineString format that we use into a collection of Points for the d3 animation
+    function linestringToPoint(coll){
+        // getting point data from list
+        var coorList = coll.features[0].geometry.coordinates;
+        var featureList = [];
+        var len = coorList.length;
+
+        for(i = 0; i < len; i ++){
```

Comment:
This is my taste, but I think coorList.map(function (c) { return turf.point(c); }) is more readable.